3604/3/config/environment.rb

```diff
@@ -0,0 +1,10 @@
+require 'bundler'
+Bundler.require
+require 'rack-flash'
+
+ActiveRecord::Base.establish_connection(
+  adapter: 'sqlite3',
+  database: 'foo.sqlite3'
```

Comment:
not sure that it is good name for database
you can set it based on environment (development, staging, production). Check RACK_ENV variable for this. You can leave development as a default